src/anders_karlsen_ex/ex05/bounded_sim.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Anders Karlsen'
+__email__ = 'anderska@nmbu.no'
+
+
+from anders_karlsen_ex.ex05.walker_sim import Walker, Simulation
+import random
+
+
+class BoundedWalker(Walker):
+    def __init__(self, start, home, left_limit, right_limit, **kwargs):
+        """
+        Initialise the walker
+
+        Arguments
+        ---------
+        start : int
+            The walker's initial position
+        home : int
+            The walk ends when the walker reaches home
+        left_limit : int
+            The left boundary of walker movement
+        right_limit : int
+            The right boundary  of walker movement
+        """
+        super().__init__(start, home, **kwargs)
+        self.left_limit = left_limit
+        self.right_limit = right_limit
+
+    def move(self):
+        throw = random.randint(1, 2)
+        if self.position == self.left_limit and throw == 1:
+            pass
+        else:
+            self.position += 1 if throw == 2 else -1
+            self.number_of_steps += 1
+
+
+class BoundedSimulation(BoundedWalker, Simulation):
```

Comment:
No need for multiple inheritance here, it would be better to just inherit the Simulation class here
And implement the move method here